src/DotNetStatus/Common/Helpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Fx.Portability;
+using Newtonsoft.Json;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace DotNetStatus.Common
+{
+    public static class Helpers
+    {
+        private static HttpClient _client;
+
+        static Helpers()
+        {
+            _client = CreateHttpClient();
+
```

Comment:
I don't understand these helpers. Why a static class? Please make it an instance class with a better name.